skyportal/handlers/api/annotation.py

```diff
@@ -110,51 +115,6 @@ def post(self):
             return self.error("Input `origin` must begin with alphanumeric/underscore")
 
         annotation_data = data.get("data")
-
-        # Ensure user/token has access to parent source
```

Comment:
So what happens if the user doesn't have access to the source? it fails upon trying to post to the database?

Reply:
Yes